java.md

```diff
@@ -26,11 +26,21 @@ performance on workloads with light to moderate lock-contention: improved spin-l
 enhanced implementation of `Thread.onSpinWait()` on Graviton2.
 
 ### Java JVM Options
-There are numerous options that control the JVM and may lead to better performance. Three that
-have shown large (1.5x) improvements in some Java workloads are eliminating tiered compilation
-and restricting the size of the code cache which allows the Graviton2 cores to better predict
-branches. These are helpful on some workloads but can hurt on others so testing with and without
-them is essential: `-XX:-TieredCompilation -XX:ReservedCodeCacheSize=64M -XX:InitialCodeCacheSize=64M`.
+There are numerous options that control the JVM and may lead to better performance.
+
+- Three flags `-XX:-TieredCompilation -XX:ReservedCodeCacheSize=64M -XX:InitialCodeCacheSize=64M`
+have shown large (1.5x) improvements in some Java workloads.
+The flags are eliminating tiered compilation and are restricting the size of
+the code cache which allows the Graviton2 cores to better predict branches.
```

Comment:
This is also relevant to Graviton3, so should we just drop the 2?

Reply:
Correct.  I will push a new change.

Reply:
I fixed a similar issue a few lines above.  We may need to review other files as well.